joinfaces-autoconfigure/src/test/java/org/joinfaces/autoconfigure/primefaces/PrimefacesPropertiesIT.java

```diff
@@ -53,97 +53,116 @@
 
 	@Test
 	public void testPrivateCaptchaKey() {
-		assertThat(this.primefaces4_0Properties.getPrivateCaptchaKey()).isEqualTo("myPrivateCaptchaKey");
```

Comment:
Let's keep the formatting changes in this class out of this PR.

Reply:
I'll take care next time. I've changed testTheme too.